lib/index.js

```diff
@@ -25,4 +54,22 @@ const setCurrentWeather = (weatherInfo) => {
   $('#current-state').text(currentWeather.state);
   $('#current-date').text(currentWeather.date);
   $('#current-icon').text('').append(currentWeather.icon);
+};
+
+const setBackgroundImage = (imageInfo) => {
```

Comment:
I would suggest not changing the css directly in your JavaScript file to better follow single responsibility principle. This way all of your styling is all contained in your css files and is easier to read and debug. Instead, I would look into using [.addClass()](https://api.jquery.com/addclass/) and [.removeClass()](https://api.jquery.com/removeclass/) to your HTML elements and then putting your styling for those classes in your css files.

Reply:
This is a really good point, thank you! I've got another PR open right now and will make this change in there.

Reply:
@kaohman question about this actually - since the URL of the image changes dynamically depending on what image my API request returns, do you know if I am able to use `addClass` but also somehow pass it the attribute of the url it should be using in this particular instance?

Reply:
Ahh okay, your original way might be the best then, if the workaround was difficult. Background images are tricky. I'm so used to React now where you are set your HTML directly in your JS files so it's more straightforward to create an `<img>` tag and set the URL src. That could be another option, to append a new `<img>` HTML element to the DOM and style it as `position: relative` to overlay other elements on top of it.